db/models/points.js

```diff
@@ -85,6 +86,7 @@ class Service extends BaseService {
       record.hash = hash.encodedString
       record.coordinates = this.makeCoordinate(point.longitude, point.latitude);
       record.time = new Date(point.time);
+      record.duration = point.duration;
```

Comment:
We need to make sure we're capturing `duration` up in `createRedactedPoint` as well.